.github/workflows/e2e.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node: ['18', '20', '22']
-        bundler: ['webpack', 'vite']
+        bundler: ['webpack', 'farm', 'vite']
```

Comment:
Could you help to put them in alphabetical order?

Reply:
Done, thank you